app/src/main/res/layout/recent_patients.xml

```diff
@@ -7,7 +7,8 @@
   <androidx.recyclerview.widget.RecyclerView
     android:id="@+id/recent_patient_recyclerview"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content" />
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="@dimen/spacing_96" />
```

Comment:
Hey, I don't think this is what we wanted. This margin will be persistent. We want the extra spacing to be present after the last item.

Reply:
Oh wait, I just realized that because this list isn't scrollable, margins == spacing after the last item. 🤦‍♂️ 

Can we still do it the correct way please? 

```xml
android:paddingBottom="@dimen/spacing_96"
android:clipChildrenToPadding=false
```

Reply:
I don't understand. How is it any different?

Reply:
Marging: spacing between the RecyclerView and the screen bottom. This spacing will always be visible.
Padding + clipChildren=false: extra spacing *after the last item*. This spacing will only be visible when you reach the bottom of the list.

Of course, this difference gets nullified because of an external factor i.e., the recent patient list doesn't scroll in home screen. It's height is set to `wrap_content`.

Reply:
If it helps: `Padding + clipChildren=false` will act as a footer.

Reply:
Let me try giving you some screenshots for comparison.

Reply:
Here you go. With margin: 

![Screenshot_1552564740](https://user-images.githubusercontent.com/2387680/54355302-c0718700-467e-11e9-9d83-ade1e9bd2cc6.png)

Reply:
Can we do this offline instead of here?
This looks strange for a PR feedback.

Reply:
Also, I have made the changes you asked.